src/build-2024/06 - Modifying Kernel Behavior with Dependency Injection/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Http;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.OpenAI;
+using Microsoft.SemanticKernel.Plugins.Web.Bing;
+
+var openAIChatCompletionModelName = "gpt-4-turbo"; // this could be other models like "gpt-4-turbo".
+
+var builder = Kernel.CreateBuilder();
+
+// injecting services to the kernel such as logging, http client, redaction.
+// builder.Services.AddLogging(b => b.AddConsole().SetMinimumLevel(LogLevel.Trace));
+
+builder.Services.ConfigureHttpClientDefaults(b =>
+{
+    b.AddStandardResilienceHandler();
+    b.RedactLoggedHeaders(["Authorization"]);
+});
+builder.Services.AddRedaction();
```

Comment:
This redaction stuff is only relevant if the AddLogging above is uncommented.

Reply:
@JakeRadMSFT  hence  we do not need a separate "Add Logging" project, we can merge it here